### DISCUSSION TITLE
To create custom .Net templates using Clean Architecture solution

### DISCUSSION THREAD

**Discussion by jithinkoduppana** [jasontaylordev/CleanArchitecture#424] [Q&A]
Hi Jason,

I have created a Clean Architecture Template based on your video and github code. 
I feel it very great to implement.Thank You very much. 
I need one help in creating the template based on the Clean Architecture solution.(same like your ca.sln)
code download link : https://github.com/jasontaylordev/CleanArchitecture/tree/main/src
Jason Taylors video link : https://www.youtube.com/watch?v=dK4Yb6-LxAk

**Comment by MichaelHochriegl** [comment]
Sorry, but I don't really get your problem. Could you maybe rephrase/define your question a little bit?

**Reply by jithinkoduppana** [reply]
I need to create custom template based on Clean Architecture solution.
Say for example I have replaced the WebUI(Presentation) with a WebAPI and used Azure SQL for database connection and created one template
Similarly I have created another Clean Architecture Template by replacing Presentation layer with a Console Application and used Azure Cosmos DB as backend.
My team mates need to use these templates.
So I need to create different templates like this and my team mates need to work on that.(same like Jason Taylor created the ca -sln template and we used it)
I need the steps in how to create the template
